.buildkite/scripts/build_and_upload_images.sh

```diff
@@ -36,7 +37,28 @@ echo "--- Building all ${IMAGE_TAG} images"
 # all the buildx file introspection (and thus build-target awareness)
 # localised here.
 make build-image-prerequisites
-docker buildx bake --file="${BUILDX_BAKE_FILE}" --progress "plain" --push "${BUILDX_TARGET}"
+
+# Build targets
+docker buildx bake --file="${BUILDX_BAKE_FILE}" --progress "plain" "${BUILDX_TARGET}"
+
+# Cloudsmith may be having trouble with `buildx --push`, so, experimenting with
+# uploading each one individually.
+{
+    # These are sans-tag
+    mapfile -t image_names < <(
```

Comment:
here's the output of `--print` btw - it notably doesn't include any of the variables for some reason

https://gist.github.com/wimax-grapl/34cd049cdeaed23f0b1f44c95135d709

Reply:
well that's with --print --push but yeah